assets/js/custom/TranslateProgram.js

```diff
@@ -0,0 +1,135 @@
+/* eslint-env jquery */
+/* global Translation */
+/* global ByLineElementContainer */
+
+// eslint-disable-next-line no-unused-vars
+class TranslateProgram extends Translation {
```

Comment:
The TranslateProgram class is similar to the TranslateComments class, but handles the translation function specific to the program section. It contains listeners for the translate and remove translation button for the program. Then it contains methods that set, open, and close the translations for the title, description, and credits of the project.